mailserver/mailserver.go

```diff
@@ -244,20 +244,40 @@ func (s *WMailServer) DeliverMail(peer *whisper.Peer, request *whisper.Envelope)
 	}
 	if s.exceedsPeerRequests(peer.ID()) {
 		requestErrorsCounter.Inc(1)
+		log.Error("Peer exceeded request per seconds limit", "peerID", peer.ID())
+		err := s.sendHistoricMessageErrorResponse(peer, request, fmt.Errorf("rate limit exceeded"))
+		if err != nil {
```

Comment:
if you rebase from `develop` we should have a `historicResponseErrorsCounter` metric that we can increment here

Reply:
you mean, inside this function?

Reply:
yes but I think the rebase or merge should already fix it:

https://github.com/status-im/status-go/blob/develop/mailserver/mailserver.go#L264